stdlib/Markdown/src/parse/parse.jl

```diff
@@ -1,5 +1,12 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    MD
+
+`MD` represents a Markdown document. Note that the `MD` constructor should not generally be
+used directly, since it constructs the internal data structures.   Instead, you can
```

Comment:
Is the use of triple spaces to separate the sentences intentional? It doesn't seem to be used elsewhere in this PR. :thinking:

Reply:
Probably an accident, though some people reflexively type two spaces after every sentence.  In any case, it's harmless, since Julia's Markdown parser ignores the additional whitespace:
```jl
julia> md"Foo.      Bar."
  Foo. Bar.
```